### ISSUE TITLE
Master branch should change name to main

### ISSUE THREAD

**Issue by pbiology** [genomic-medicine-sweden/gms-artic#22]
**What needs to be done:**
The name "Master" isn't in use in best practices for GitHub anymore.

**Suggestions on how to get it done:**
Change it to "Main"

**What are the arguments for getting it done:**
Following best practices

**Task is considered finished when:**
Name is changed!  :)

**Comment by JD2112** [comment]
@pbiology I create the "main" branch. Shall I set it to default? 
@sofstam If the default branch changes to "main", will it impacts on the GH actions?

**Comment by sofstam** [comment]
@JD2112 Yes, it will impact. The GH action `pull_request.yml`  should be changed to:

```
on:
  pull_request:
    branches:
    - main
```
